.github/workflows/build.yml

```diff
@@ -20,8 +19,13 @@ jobs:
         with:
           toolchain: stable
           default: true
-      - name: Build   
+      - name: Build
+        uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: --all
+      - name: Build async rio
         uses: actions-rs/cargo@v1
         with:
           command: build
-          args: --all
\ No newline at end of file
+          args: --features async-io-rio
```

Comment:
Same should be done for tests: https://github.com/qoollo/pearl/blob/master/.github/workflows/tests.yml#L27

First, run test with default features. Then add new step to run them with `async-io-rio` feature